common/src/errors.rs

```diff
@@ -70,6 +72,21 @@ error_chain! {
             description("missing debug info")
             display("missing debug info: {}", message)
         }
+        /// Raised for JSON parse errors.
```

Comment:
Need feature flags here?

Reply:
Nope, I want the error types to always exist.